language/tools/test_generation/src/bytecode_generator.rs

```diff
@@ -36,6 +40,22 @@ enum BytecodeType {
 
     /// Instructions that take an `AddressPoolIndex`
     AddressPoolIndex(AddressPoolIndexToBytecode),
+
+    /// Instructions that take a `ByteArrayPoolIndex`
+    ByteArrayPoolIndex(ByteArrayPoolIndexToBytecode),
+}
+
+/// Abstraction for change to the stack size
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
Use Copy rather than Clone

Reply:
Unfortunately not, as StackEffect is used in the BytecodeGenerator struct that uses both Bytecode and StdRng which don't implement copy.

Reply:
Can't you then implement both Copy and Clone?

Reply:
Yes, we could but Copy and Clone are essentially the same besides that Copy is guaranteed to be inexpensive. However, the `StackEffect` is only cloned in one place so I don't think it will make much of a difference in performance. In any case I'll make the change.